src/components/PersistentDrawer.js

```diff
@@ -108,26 +108,13 @@ const styles = theme => ({
 });
 
 const optionKeyLabels = {
-  target: 'Allowable Target',
-  strict: 'Enforce code cleanliness standards',
-  minify: 'Reduce size of Compiled Python',
-  line_numbers: 'Show line numbers',
-  keep_lines: 'Keep line numbers',
-  no_tco: 'No TCO',
+  target: 'Allowable Targets',
```

Comment:
Change to just Python version target as @jcruz said.

Reply:
Done